klippy/webhooks.py

```diff
@@ -320,6 +331,26 @@ def get_status(self, eventtime):
         state_message, state = self.printer.get_state_message()
         return {'state': state, 'state_message': state_message}
 
+    def call_remote_method(self, method, **kwargs):
+        if method not in self._remote_methods:
+            self.printer.lookup_object('gcode').respond_info(
+                "Remote method '%s' not registered" % (method))
```

Comment:
FWIW, I'm not sure `respond_info()` is a good way to report a problem.  I can't see regular users being able to find that info or make sense of it in the gcode output.

-Kevin

Reply:
That makes sense.  I pushed some commits that raise a CommandError instead.  For `action_call_remote_method` I handle the error and log it, as I don't think we would want this particular error aborting a print.  If you think they look okay I'll squash them into the original commits.